### DISCUSSION TITLE
Failed assertions with Unreal Engine 5

### DISCUSSION THREAD

**Discussion by ssuukk** [nxrighthere/UnrealCLR#253] [Help] [answered]
Following assertions are failing:

	static_assert(ControllerHand::ControllerHand_Count == ControllerHand(17), "Invalid elements count of the [ControllerHand] enumeration"); (there's 18)
	static_assert(PixelFormat::PF_MAX == PixelFormat(UNREALCLR_PIXEL_FORMAT), "Invalid elements count of the [PixelFormat] enumeration"); (probably 85)
	static_assert(sizeof(Bounds) == 28, "Invalid size of the [Bounds] structure");

**Comment by nxrighthere** [comment, marked as answer]
Please, see https://github.com/nxrighthere/UnrealCLR/discussions/243 and https://github.com/nxrighthere/UnrealCLR/pull/248.
